build.gradle

```diff
@@ -41,6 +39,12 @@ ext {
 group = projectGroupId
 version = currentVersion
 
+sourceSets.main.java.srcDirs = []
+sourceSets.main.groovy.srcDirs += ["src/main/java"]
+
+sourceCompatibility = '1.8'
+targetCompatibility = '1.8'

```

Comment:
`targetCompatibility` is probably [redundant](http://gradle.org/docs/current/userguide/java_plugin.html#N1233C).